packages/aws-cdk/test/integ/cli/aws-helpers.ts

```diff
@@ -138,6 +154,52 @@ export async function retry<A>(operation: string, deadline: Date, block: () => P
   }
 }
 
+/**
+ * Make a deadline for the `retry` function relative to the current time.
+ */
+retry.forSeconds = (seconds: number): Date => {
+  return new Date(Date.now() + seconds * 1000);
+};
+
+/**
+ * Annotate an error to stop the retrying
+ */
+retry.abort = (e: Error): Error => {
+  (e as any).abort = true;
```

Comment:
Minor comment here, in the same vein as the function name, although less serious (`abort` might mean "this was already aborted") - I would maybe rename this `aborted` (or even `shouldRetry`)...? Do what you will with this comment, this isn't super crucial.